ground/src/main/java/com/google/android/ground/ui/surveyselector/SurveySelectorViewModel.kt

```diff
@@ -55,38 +53,41 @@ internal constructor(
   private val userRepository: UserRepository,
 ) : AbstractViewModel() {
 
-  // TODO(#2081): Expose non-mutable state flow.
-  val surveyListState: MutableStateFlow<State?> = MutableStateFlow(null)
-  private val _errorFlow: MutableSharedFlow<Int?> = MutableSharedFlow()
-  val errorFlow: Flow<Int> = _errorFlow.asSharedFlow().filterNotNull()
+  private val _uiState: MutableStateFlow<UiState?> = MutableStateFlow(null)
+  val uiState: StateFlow<UiState?> = _uiState.asStateFlow()
```

Comment:
rather than null as the initial value, we can keep some default `UiState`? WDYT?